plugins/permissions/handlers.go

```diff
@@ -34,6 +35,7 @@ func (p *permissions) getPermission() http.HandlerFunc {
 func (p *permissions) postPermission(opts ...permission.Options) http.HandlerFunc {
```

Comment:
@bietkul Here, you aren't using `opts` at all. Have you tested this with [POST /role/:role](https://arc-api.appbase.io/?version=latest#441f1f65-bbcc-467a-88fb-c14926be7036)?